pyreq2rpm/pyreq2rpm.py

```diff
@@ -72,6 +75,9 @@ def convert_compatible(name, operator, version_id):
     version = RpmVersion(version_id)
     if len(version.version) == 1:
         return 'Invalid version'
+    if version.is_legacy():
```

Comment:
Should this be checked before `len(version.version)`?

Reply:
Sure thing